connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResource.java

```diff
@@ -341,7 +341,15 @@ private void checkAndPutConnectorConfigName(String connectorName, Map<String, St
                     // this gives two total hops to resolve the request before giving up.
                     boolean recursiveForward = forward == null;
                     RequestTargetException targetException = (RequestTargetException) cause;
-                    String forwardUrl = UriBuilder.fromUri(targetException.forwardUrl())
+                    String forwardedUrl = targetException.forwardUrl();
+                    if (forwardedUrl == null) {
+                        // the target didn't know of the leader at this moment.
+                        // we don't, it probably means that a rebalance has taken place.
```

Comment:
this second comment line doesn't read very clearly for me. 
who's "we" in this case? 

wouldn't `this probably means that a rebalance has taken place` be as good?

Reply:
Ack.